cmd/juju/commands/run_test.go

```diff
@@ -274,6 +274,28 @@ func (s *RunSuite) TestRunForMachineAndUnit(c *gc.C) {
 	c.Check(cmdtesting.Stdout(context), gc.Equals, buff.String())
 }
 
+func (s *RunSuite) TestRunForMissingUnit(c *gc.C) {
```

Comment:
We don't really need this test since AFAIK there's no bespoke error handling in the api or cli layers - they just pass any error straight through. The code changes and tests are all in the apiserver layer. This test doesn't really test anything different to what would already be in this suite, unless there's no test for any error handling to start with, in which case it would just be TestRunWithErrorResult or something.